java/src/main/java/utility/CommonContext.java

```diff
@@ -265,6 +267,26 @@ public static GenericRecord deserialize(Schema schema, ByteString payload) throw
         return reader.read(null, decoder);
     }
 
+    public static void processAndPrintChangedFields(Schema writerSchema, GenericRecord record) throws IOException {
```

Comment:
I see that we print only the changed fields. Is it possible to add a comment similar to:

// This method expands the changedFields bitmap field in ChangeEventHeader. To expand the other bitmap fields, diffFields and nulledFields, modify this code.